pkg/controller/topology.go

```diff
@@ -183,6 +177,7 @@ func aggregateTopologies(
 	if err != nil {
 		return nil, err
 	}
+	// TODO: use informers
```

Comment:
TODO(ISSUENUMBER): 

Please create an issue for tracking :)

Reply:
#144 already created :-)